README.md

```diff
@@ -16,52 +16,53 @@ To add the driver to your project, add the following Maven dependency:
 
-### JDBC Driver class
+### Driver Class (Reiterated)
 
-Use `com.salesforce.datacloud.jdbc.DataCloudJDBCDriver` as the driver class name for the JDBC application.
+Use `com.salesforce.datacloud.jdbc.DataCloudJDBCDriver` when loading the driver in your JDBC application. For example:
 
-### Authentication
+```java
+Class.forName("com.salesforce.datacloud.jdbc.DataCloudJDBCDriver");
+```
 
-We support three of the [OAuth authorization flows][oauth authorization flows] provided by Salesforce.
-All of these flows require a connected app be configured for the driver to authenticate as, see the documentation here: [connected app overview][connected app overview].
-Set the following properties appropriately to establish a connection with your chosen OAuth authorization flow:
+### Authentication
 
-| Parameter    | Description                                                                                                          |
-|--------------|----------------------------------------------------------------------------------------------------------------------|
-| user         | The login name of the user.                                                                                          |
-| password     | The password of the user.                                                                                            |
-| clientId     | The consumer key of the connected app.                                                                               |
-| clientSecret | The consumer secret of the connected app.                                                                            |
-| privateKey   | The private key of the connected app.                                                                                |
-| coreToken    | OAuth token that a connected app uses to request access to a protected resource on behalf of the client application. |
-| refreshToken | Token obtained from the web server, user-agent, or hybrid app token flow.                                            |
+The driver supports several OAuth authorization flows provided by Salesforce (see [OAuth authorization flows](https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_flows.htm&type=5)). You'll need to configure a connected app (see [Connected App Overview](https://help.salesforce.com/s/articleView?id=sf.connected_app_overview.htm&type=5)).
 
+The following table outlines the properties for each flow:
 
-#### username and password authentication:
+| Property       | Description                    | Authentication Flow              |
+| -------------- | ------------------------------ | -------------------------------- |
+| `user`         | Salesforce username.           | Username/Password                |
+| `password`     | Salesforce password.           | Username/Password                |
+| `clientId`     | Connected app consumer key.    | All                              |
```

Comment:
I think we should put the "All" entry to the top, instead of somwhere in the middle of the table

```suggestion
| `clientId`     | Connected app consumer key.    | All                              |
| `user`         | Salesforce username.           | Username/Password                |
| `password`     | Salesforce password.           | Username/Password                |
```